<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfigureCam</class>
 <widget class="QDialog" name="DialogConfigureCam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>944</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="buttonSaveConfig">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonLoadConfig">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonResetDefaults">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="labelPixelClock">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLabel" name="labelBlueGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelFrameRate">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="labelExposure">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="labelMasterGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="labelRedGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="labelGreenGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="2">
            <widget class="QSlider" name="sliderPixelClock">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QSlider" name="sliderFrameRate">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="1" colspan="2">
            <widget class="QSlider" name="sliderExposure">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="1" colspan="2">
            <widget class="QSlider" name="sliderMasterGain">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="9" column="1" colspan="2">
            <widget class="QSlider" name="sliderRedGain">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="11" column="1" colspan="2">
            <widget class="QSlider" name="sliderGreenGain">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="13" column="1" colspan="2">
            <widget class="QSlider" name="sliderBlueGain">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="curPixelClock">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="curFrameRate">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLabel" name="curExposure">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QLabel" name="curMasterGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="9" column="3">
            <widget class="QLabel" name="curRedGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="11" column="3">
            <widget class="QLabel" name="curGreenGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="13" column="3">
            <widget class="QLabel" name="curBlueGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="minPixelClock">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="maxPixelClock">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="minFrameRate">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="maxFrameRate">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="minExposure">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="maxExposure">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="minMasterGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QLabel" name="maxMasterGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLabel" name="minRedGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QLabel" name="maxRedGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="minGreenGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QLabel" name="minBlueGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="14" column="2">
            <widget class="QLabel" name="maxBlueGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="12" column="2">
            <widget class="QLabel" name="maxGreenGain">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelWidth">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editSizeX">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelHeight">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="editSizeY">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="preview">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>480</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>480</width>
             <height>1000</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRefresh">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkAutoRefresh">
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
